.github/workflows/02-e2e-screen-reader.yml

```
@@ -26,6 +26,14 @@ jobs:
         uses: actions/checkout@v4
 
       - name: 🦮 Guidepup Setup
+        id: setup
+        continue-on-error: true
+        uses: guidepup/setup-action@0.17.2
+        with:
+          record: true
+
+      - name: 🦮 Guidepup Setup fallback
```

```suggestion
      - name: 🦮 Guidepup Setup retry
```

It's mainly a retry, isn't it? Or am I getting something wrong?